h/views/activity.py

```diff
@@ -54,18 +53,22 @@ def search(request):
         'page': paginate(request, result.total, page_size=page_size),
     }
 
-
 @view_config(route_name='activity.group_search',
              request_method='GET',
              renderer='h:templates/activity/search.html.jinja2')
 def group_search(request):
     if not request.feature('search_page'):
         raise httpexceptions.HTTPNotFound()
 
+    opts = {}
+
     result = search(request)
 
     pubid = request.matchdict['pubid']
 
+    opts['search_groupname'] = request.matchdict['pubid']
```

Comment:
This says group name but you're using the pubid here, from the user's POV, they should see the name rather than the pubid on the lozenge.